docs/oadp-backups.md

```diff
@@ -0,0 +1,225 @@
+              - -c
+              - |
+                cd /tmp
+                curl -L -O https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-linux-amd64
+                mv jq-linux-amd64 jq
+                chmod +x jq
+                oc scale -n openshift-mta deployment rhsso-operator --replicas=0
+                oc wait -n openshift-mta po -l name=rhsso-operator --for=delete
+                oc scale -n openshift-mta deployment mta-operator --replicas=0
+                oc wait -n openshift-mta po -l name=mta-operator --for=delete
+                oc scale -n openshift-mta statefulset keycloak --replicas=0
+                oc wait -n openshift-mta po -l app=keycloak --for=delete
+                oc scale -n openshift-mta deployment mta-keycloak-postgresql-15 --replicas=0
+                oc wait -n openshift-mta po -l app.kubernetes.io/name=mta-keycloak-postgresql-15 --for=delete
+                oc scale -n openshift-mta deployment mta-ui --replicas=0
+                oc wait -n openshift-mta po -l app.kubernetes.io/name=mta-ui --for=delete
+                oc scale -n openshift-mta deployment mta-hub --replicas=0
+                oc wait -n openshift-mta po -l app.kubernetes.io/name=mta-hub --for=delete
+                export BACKUP_DATE=$(date +%Y-%m-%d-%H-%M-%S)
+                cat << EOF | oc apply -f -
+                kind: Backup
+                apiVersion: velero.io/v1
+                metadata:
+                  name: backup-$BACKUP_DATE
+                  namespace: openshift-adp
+                spec:
+                  csiSnapshotTimeout: 30m0s
+                  defaultVolumesToFsBackup: false
+                  includedNamespaces:
+                  - openshift-mta
+                  itemOperationTimeout: 30h0m0s
+                  snapshotMoveData: true
+                  backupLocation: velero-1
+                  storageLocation: velero-1
+                  ttl: 720h0m0s
+                  volumeSnapshotLocations:
+                  - velero-1
+                EOF
+                while [[ $(oc get -n openshift-adp backup backup-$BACKUP_DATE -o json | ./jq -r '.status.phase') != "Completed" ]]; do sleep 5; done
+                oc scale -n openshift-mta statefulsets --all --replicas=1
+                oc scale -n openshift-mta deployments --all --replicas=1
+          restartPolicy: OnFailure
+          serviceAccount: job-runner
+```
+
+## Running Restores
+
+Running a restore of the namespace is not much different than a restore for any other application outside of a minor considerations.
+
+The RHSSO operator will fail to reconcile if we restore either the `prometheusrules.monitoring.coreos.com` or `servicemonitors.monitoring.coreos.com` resource due to missing ownerRefs. In turn the MTA Operator will also fail to reconcile. We can avoid this problem by excluding the resources from the restore and allowing the RHSSO. All we need to do is pick a backup to restore and create the restore resource:
```

Comment:
If we don't need these resources then maybe just add them to `excludedResources` under backup spec and exclude them from backup ?